.github/workflows/release.yml

```
@@ -0,0 +1,126 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  tag:
+    name: Check and Tag
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Create tag
+        id: tag
+        uses: butlerlogic/action-autotag@1.1.1
+        with:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          strategy: regex
+          tag_prefix: "v"
+          root: .version
+          regex_pattern: version=([0-9\.]{5}([-\+][\w\.0-9]+)?)
+
+    outputs:
+      tag: ${{ steps.tag.outputs.tagname }}
+      version: ${{ steps.tag.outputs.version }}
+
+
+  publish:
+    name: Publish
+    needs: tag
+    if: ${{ needs.tag.outputs.tag == ''}}
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: "on"
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: 1.14.x
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build All
+        shell: bash
+        env:
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+        run: |
+          SENTRY_DSN=$SENTRY_DSN make buildAll
+
+      - name: Get Changelog Entry
+        id: changelog_reader
+        uses: mindsers/changelog-reader-action@v2
+        with:
+          version: ${{ needs.tag.outputs.version }}
```

```suggestion
          version: ${{ needs.tag.outputs.tag }}
```
since you will probably put v1.0.0 in changelog and version doesn't contain v prefix